packages/react-hooks/README.md

```diff
@@ -40,3 +40,19 @@ function MyComponent() {
   return <div>{foo}</div>;
 }
 ```
+
+### `useLazyRef()`
```

Comment:
isn't this basically just `useMemo` with no dependencies?

Reply:
`useMemo` is not guaranteed to actually be the same forever (they might ditch some stuff if there is memory pressure). This one is a guarantee

Reply:
👍